packages/chatbot-server-mongodb-public/src/systemPrompt.ts

```diff
@@ -31,21 +32,35 @@ const technicalKnowledge = [
 ];
 
 const importantNotes = [
-  `ALWAYS use the ${SEARCH_TOOL_NAME} tool at the start of the conversation. Zero exceptions!`,
-  `Use the ${SEARCH_TOOL_NAME} tool after every single user message.`,
+  "User messages may be accompanied by metadata explaining where the chatbot is on the site.",
```

Comment:
my prompting intuition tells me that this should be a separate section of the prompt, rather than just putting in importantNotes.

can create new section of prompt like:

```txt
<message-metadata>
User messages may be accompanied by metadata as follows:
{{list of metadata rules}}
</message-metadata>

```